src/vuex/modules/connection.js

```diff
@@ -32,19 +32,14 @@ export default function({ apollo }) {
   const actions = {
     async checkForPersistedNetwork({ dispatch, commit }) {
```

Comment:
probably chain `checkForPersistedNetwork` after `preloadNetworks` then?

Reply:
Why? They are not logically connected. If you are about to avoid await on preloadNetworks that will no do any matter

Reply:
they apparently are logically connected if `checkForPersistedNetwork` needs the networks to have been loaded

Reply:
But many functions need the, to be loaded. Why do we need to chain only this one?